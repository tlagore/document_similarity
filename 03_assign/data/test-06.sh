#!/usr/bin/env bash
sbt --error 'set showSuccess := false' 'run data/articles_100.txt  " " 0.15 2 20 3 true true false'
