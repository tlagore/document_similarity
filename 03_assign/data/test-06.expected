Computing similarity with parameters: 
    Filename: data/articles_100.txt
    Separator: [ ]
    Minimum similarity: 0.15
    Shingle size: 2
    Hash count: 20
    Band size: 3
    Compute Jaccard similarity: true
    Compute all minHashes similarity: true
    Print hash coefficients: false

Pair                     Jaccard              minHash                  lsh
--------------------------------------------------------------------------
t1088,t5015              0.98828              1.00000              1.00000
t1297,t4638              0.98832              1.00000              1.00000
t1768,t5248              0.98780              1.00000              1.00000
t1952,t3495              0.98684              1.00000              1.00000
t2023,t980               0.98734              0.95000              0.85714
t3174,t3176                  ---              0.20000                  ---
t5293,t5294                  ---              0.15000                  ---
t5295,t5299                  ---              0.15000                  ---
t9535,t9537                  ---              0.15000                  ---
--------------------------------------------------------------------------
